src/pages/steps/CalendarStep/CalendarStep.tsx

```diff
@@ -253,8 +256,27 @@ const CalendarStep = ({
     handleChooseFixedDays();
   }, [scope, isIdle, handleChooseFixedDays]);
 
+  // scroll to calendar step after theme has been selected
+  useEffect(() => {
+    if (!scope || !type.id) return;
+    if (offerId) return;
+
+    const yOffset = -60;
+    const main = document.getElementsByTagName('main')[0];
+    const calendarContainer = document.getElementById(
+      'calendar-step-container',
+    );
```

Comment:
You could also use a ref to select the container instead of using an id.

https://beta.reactjs.org/learn/manipulating-the-dom-with-refs#example-scrolling-to-an-element

Reply:
Yes good idea